airbyte-integrations/connectors/source-oracle/src/main/resources/spec.json

```diff
@@ -4,7 +4,13 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Oracle Source Spec",
     "type": "object",
-    "required": ["host", "port", "sid", "username"],
+    "required": [
+      "host",
+      "port",
+      "sid",
+      "username",
+      "encryption"
```

Comment:
Please ensure backward compatibility with previous versions of the connector.

Reply:
> Please ensure backward compatibility with previous versions of the connector.

removed encryption from reqired fields